go.mod

```diff
@@ -5,25 +5,54 @@ go 1.22.0
 toolchain go1.22.2
 
 require (
-	k8s.io/apimachinery v0.30.0
+	k8s.io/apimachinery v0.30.3
+	k8s.io/client-go v0.29.2
```

Comment:
Does it have to be this version of the client?

Reply:
+1
Yes, it should be.

Reply:
We can all start with v0.30.3

Reply:
fixed.